src/lib/utils/parseJson.ts

```diff
@@ -0,0 +1,8 @@
+export const parseJsonOrString = (jsonString: string): Record<string, any> => {
+  try {
+    return JSON.parse(jsonString)
+  } catch (e) {
+    // Convert string to an object with a value property if parsing fails
+    return { value: jsonString }
```

Comment:
this is a bad pattern,

1) we never use the meaningless variable name `value` unless there is some industry-specific reason we have to

2) the function name makes what this does unclear

3) if something fails to parse, there are only two appropriate conventions: return null OR return a "go-style" error e.g. `return { error } | { error: null, json }`

4) the filename doesn't match the function name